### DISCUSSION TITLE
Any plans to remove hardcoded font links?

### DISCUSSION THREAD

**Discussion by han9088** [toeverything/blocksuite#6006] [General]
https://github.com/nawbc/blocksuite/blob/master/packages/blocks/src/surface-block/consts.ts#L61

Hardcode domain URL in an OSS may result in the following. Even resolving CORS issues may lead to other surprises.

![image](https://github.com/toeverything/blocksuite/assets/45007226/8cd17758-995a-49ee-9ed5-c09ef42ef14b)

**Comment by doodlewind** [comment]
Yes, we have plan to make them configurable options in block service, so we can define them in the block spec level for composing editors (https://github.com/toeverything/blocksuite/blob/master/packages/blocks/src/_specs/_specs.ts). The time for the refactoring is not decided yet, but it's supposed should be soon.
